adding_species_manuscript.tex

```diff
@@ -187,19 +187,23 @@ \section*{Abstract}
 \section*{Introduction}
     \label{introduction}
 
-Dramatic reductions in sequencing costs are enabling the generation of
-unprecedented amounts of genomic data for a huge variety of species
-\citep{Ellegren2014}. Ongoing efforts to systematically sequence life on
+%REVISION: modified beginning of paragraph (rev 1 comment)
+Population genetics allows us to answer questions across scales from deep evolutionary time to ongoing ecological dynamics,
+and dramatic reductions in sequencing costs enable the generation of
+unprecedented amounts of genomic data that can be used to address these questions \citep{Ellegren2014}.
+Ongoing efforts to systematically sequence life on
 Earth by initiatives such as the Earth Biogenome \citep{Lewin2022} and its
 affiliated project networks, such as Vertebrate Genomes
 \citep{Rhie2021}, 10,000 Plants \citep{Cheng2018} and others \citep{darwin2022sequence},
 are providing the backbone for enormous increases in the amount of population-level genomic data available for model and non-model species.
-%REVISION: TBD - add more details and citations to this sentence
```

Comment:
I tried to merge this sentence with the one that was after it about methods development. I think it conveys the point. Please check.